src/main/groovy/wooga/gradle/unity/tasks/Test.groovy

```diff
@@ -178,6 +192,23 @@ class Test extends AbstractUnityTask implements Reporting<UnityTestTaskReport> {
 
         args(buildTestArguments(unityVersion))
         super.exec()
+
+        //normalize test result file
+        normalizeTestResult()
+    }
+
+    //https://issues.jenkins-ci.org/browse/JENKINS-44072
+    protected void normalizeTestResult() {
+        DefaultArtifactVersion unityVersion = getUnityVersion(getUnityPath())
+        if ((unityVersion.majorVersion == 5 && unityVersion.minorVersion == 6)
+                || (unityVersion.majorVersion == 2017 && unityVersion.minorVersion == 1)) {
```

Comment:
Since no future fix is really planned, I would actually check the other way around!?
Or, since you do check for JUnit version later, always call normalize!

Reply:
Yeah I thought about it. But this plugin needs an update for any future Unity version anyway. But yeah I could get rid of the version check here.